api/shield_net/services/__init__.py

```diff
@@ -7,7 +8,8 @@ def post_process_content(content: str) -> str:
     :param content: The raw content generated by the LLM.
     :return: The processed content.
     """
-    # Add custom post-processing logic here if needed
+    replacePII = ReplacePII()
```

Comment:
@enggabhishek you init the class each call function.  It's not good,  pass the class instance as argument

Reply:
Updated the code as per your suggestion.